tachyon/zk/plonk/examples/BUILD.bazel

```diff
@@ -24,16 +24,46 @@ tachyon_cc_library(
 # TODO(dongchangYoo): This is failed in CI because of timeout, 60 secs.
 # Change |tachyon_cc_test| to |tachyon_cc_unittest| once CI can run within the timeout.
 tachyon_cc_test(
-    name = "examples_unittests",
-    srcs = [
-        "simple_circuit_unittest.cc",
-        "simple_lookup_circuit_unittest.cc",
-        "simple_lookup_v1_circuit_unittest.cc",
-        "simple_v1_circuit_unittest.cc",
+    name = "simple_circuit_test",
+    srcs = ["simple_circuit_unittest.cc"],
+    deps = [
+        ":circuit_test",
+        ":simple_circuit",
+        "//tachyon/zk/plonk/halo2:pinned_verifying_key",
+        "//tachyon/zk/plonk/keys:proving_key",
+        "//tachyon/zk/plonk/layout/floor_planner:simple_floor_planner",
     ],
+)
+
+tachyon_cc_test(
+    name = "simple_v1_circuit_test",
+    srcs = ["simple_v1_circuit_unittest.cc"],
     deps = [
         ":circuit_test",
         ":simple_circuit",
+        "//tachyon/zk/plonk/halo2:pinned_verifying_key",
+        "//tachyon/zk/plonk/keys:proving_key",
+        "//tachyon/zk/plonk/layout/floor_planner/v1:v1_floor_planner",
+    ],
+)
+
+tachyon_cc_test(
+    name = "simple_lookup_circuit_test",
+    srcs = ["simple_lookup_circuit_unittest.cc"],
+    deps = [
+        ":circuit_test",
+        ":simple_lookup_circuit",
+        "//tachyon/zk/plonk/halo2:pinned_verifying_key",
+        "//tachyon/zk/plonk/keys:proving_key",
+        "//tachyon/zk/plonk/layout/floor_planner:simple_floor_planner",
+    ],
+)
+
+tachyon_cc_test(
+    name = "simple_lookup_v1_test",
+    srcs = ["simple_lookup_v1_circuit_unittest.cc"],
+    deps = [
+        ":circuit_test",
```

Comment:
Please sort targets alphabetically. They should be located before `simple_v1_circuit_test`.